frontend/src/containers/home.tsx

```diff
@@ -71,6 +71,13 @@ const HomePage = () => {
     return themeContext.theme === LIGHT ? lightDeveloper : darkDeveloper;
   };
 
+  const [ourProjects, setOurProjects] = useState([]);
+  fetch("https://raw.githubusercontent.com/osscameroon/project-ideas/master/prod-projects.json")
```

Comment:
Can we use something similar to the `useQuery` hook we use on line `53` and `59` ? 

It seems that this fetch will be triggered whenever this page is loaded this will cause the call ta hit the rate limits even quicker than we might expect